core/src/components/item-divider/item-divider.md.vars.scss

```diff
@@ -14,7 +14,7 @@ $item-divider-md-color:              $text-color-step-600 !default;
 $item-divider-md-background:         $background-color !default;
 
 /// @prop - Font size for the divider
-$item-divider-md-font-size:          0.875rem !default;
+$item-divider-md-font-size:          14px !default;
```

Comment:
This seems to be reverting your [work](https://github.com/ionic-team/ionic-framework/pull/27947/files#diff-6bd03c0d8ac5b5a7f71d9d20ffe7069d6606a73701153130d282938ad21dd3aaR17) on `item-divider`. Is this intentional?

Reply:
It is still using `dynamic-font` I just didn't put it in the variable since the font size in px is reused: https://github.com/ionic-team/ionic-framework/pull/27989/files#diff-40f8af9d7728f2c5784d79c2a2ec084752d277e28b30a31c6ab66d65cc206ce7L17

Reply:
That makes sense. 🤦‍♀️